x/twap/logic_test.go

```diff
@@ -134,7 +135,7 @@ func (s *TestSuite) TestNewTwapRecord() {
 	}
 	for name, test := range tests {
 		s.Run(name, func() {
-			twapRecord, err := twap.NewTwapRecord(s.App.GAMMKeeper, s.Ctx, test.poolId, test.denom0, test.denom1)
+			twapRecord, err := twap.NewTwapRecord(s.App.PoolManagerKeeper, s.Ctx, test.poolId, test.denom0, test.denom1)
```

Comment:
This test is initialized with a balancer pool only. I think we need to refactor this test to also include concentrated pool as well.

Reply:
https://github.com/osmosis-labs/osmosis/pull/4299/commits/1d1761106a70069bfd63ffe1427cbad0ebf6e9ca